hvplot/converter.py

```diff
@@ -286,7 +292,7 @@ class HoloViewsConverter:
 
     _op_options = [
         'datashade', 'rasterize', 'x_sampling', 'y_sampling',
-        'aggregator'
+        'aggregator', "aggregation_threshold"
```

Comment:
Nit pick, it's probably best to keep the quotes consitant and in this case use single quotes.

Additional question: do you think `downsample` (recently added option) should be added in this list? It could be an oversight.